### PR TITLE
feat(front): add file upload at publication page

### PR DESCRIPTION
**Pull request by asf-algoo**
https://github.com/tracim/tracim/issues/4255

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by gbonaspetti — APPROVED** [review]

File: frontend/src/css/Publications.styl [hunk 1]
```diff
@@ -21,13 +21,22 @@
...
+      justify-content space-between
       margin-top 10px
       order 3
-      &__advancedEdition
-        margin-inline-end auto
-        font-size fontSizeSmall
       > :last-child
         margin-inline-start auto
+      &__submit .iconbutton__icon
+        margin-right 8px
```
[the remark this answers is not in the document]
Comment (Skylsmoi) [reply frontend/src/css/Publications.styl]: fixed

**Review by raphj — APPROVED** [review]

**Comment by Skylsmoi** [comment]
regarding tests: I have added this PR to the issue to track tests that are skip to finish 3.7 and 3.8 early

**Comment by Skylsmoi** [comment]
For tester: test the addition of file in publication, with and without an associated text.
Also test the addition of file in a comment in publication, with and without an associated text

**Comment by gbonaspetti** [comment]
Tested on branch

**Pull request merged**